group-coordinator/src/test/java/org/apache/kafka/coordinator/group/GroupMetadataManagerTest.java

```diff
@@ -2798,8 +2798,6 @@ public void testRevocationTimeoutLifecycle() {
                 .setGroupId(groupId)
                 .setMemberId(memberId1)
                 .setMemberEpoch(1)
-                .setRebalanceTimeoutMs(90000)
```

Comment:
why did we remove this?

Reply:
it was not needed so removed it.

Reply:
it must be in first request or when changed.